Eastern-Research-Group/mywaterway
# Feature/update cypress tests

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3646931

## Main Changes:
* Upgraded cypress and other modules at the app level.
* Fixed a few cypress tests. 
  * The "Clicking a Discharger accordion item expands it" Identified Issues tab and "Navigating to a sub-tab selection that has no data results" State Water Quality Overview tab tests were issues prior to the cypress upgrade. The other tests were broken because of the cypress upgrade.
  * Note: The latest version of Cypress changed how `.should('not.be.visible')` works. Before you could use this as a way to check if the item was not on the DOM, now it this line is checking if the item is on the DOM and not visible. 

## Steps To Test:
1. Run `npm install`
2. Run `npm run coverage`
3. Run all cypress tests
4. Verify that all tests pass
